docs/pages/versions/unversioned/sdk/media-library.mdx

```diff
@@ -80,6 +80,12 @@ You can configure `expo-media-library` using its built-in [config plugin](/confi
         'Sets whether or not to request the `ACCESS_MEDIA_LOCATION` permission on Android.',
       default: 'false',
     },
+    {
+      name: 'granularPermissions',
+      platform: 'android',
+      description:
+        'Controls which READ_MEDIA_* permissions are added to AndroidManifest.xml. By default, all are included. Set to false to exclude them entirely (e.g. if you only use `saveToLibraryAsync`), or pass an object like { images: true, video: true } to include only specific permissions.',
```

Comment:
```suggestion
        'Controls which `READ_MEDIA_*` permissions are added to **AndroidManifest.xml**. By default, all are included. Set it to `false` to exclude them entirely (for example, if you only use `saveToLibraryAsync`), or pass an object like `{ images: true, video: true }` to include only specific permissions.'
```

Sorry, I missed this but let's sync the suggested changes to the unversioned as well.